chapters/package-information.md

```diff
@@ -225,7 +225,7 @@ EXAMPLE 2 RDF: property `spdx:supplier` in class `spdx:Package`
 
 ### 7.6.1 Description
 
-If the package identified in the SPDX document originated from a different person or organization than identified as Package Supplier (see [7.5](#3.5) above), this field identifies from where or whom the package originally came. In some cases a package may be created and originally distributed by a different third party than the Package Supplier of the package. For example, the SPDX document identifies the package [glibc][] and [Red Hat][] as the Package Supplier, but the [Free Software Foundation][FSF] is the Package Originator.
+If the package identified in the SPDX document originated from a different person or organization than identified as Package Supplier (see [7.5](#3.5) above), this field identifies from where or whom the package originally came. In some cases a package may be created and originally distributed by a different third party than the Package Supplier of the package. For example, the SPDX document identifies the package as [glibc][] and [Red Hat][] as the Package Supplier, but the [Free Software Foundation][FSF] is the Package Originator.
```

Comment:
How about:

```
For example, the SPDX document identifies the package as [glibc][] and the package supplier as [Red Hat][],
```

to make it clearer and a bit more consistent.

Reply:
I think that that’s clearer and more consistent. I force pushed a new version of the commit that uses your suggestion, but I removed my sign-off. I’m not very familiar with DCOs, but I think that I need your permission before I can sign-off on the commit again.